doc/dev/cover/index.md

```diff
@@ -0,0 +1,3 @@
+# Code Coverage
+
+This is a placeholder page which should be replaced with code coverage results.
```

Comment:
This placeholder text is intended here, right?

Reply:
Yes